index.html

```diff
@@ -86,7 +86,7 @@ <h1 class="my-4 text-5xl font-bold leading-tight">
             An Open Source Community that focuses on decentralized applications, web 3, and blockchain technologies 🚀
           </p>
           <a href="https://github.com/web3community/start-here" target="_blank" rel="noopener">
-            <button style="background: #0086D6" class="mx-auto lg:mx-0 hover:underline text-white rounded-md my-6 py-4 px-8 shadow-lg focus:outline-none focus:shadow-outline transform transition hover:scale-105 duration-300 ease-in-out">
```

Comment:
Hi, you should remove the whole class hover:underline :)

Reply:
Made the requested changes, thanks